words/words.md

```diff
@@ -93,9 +93,9 @@ Dies ist eine "S-förmige" **logistische Wachstumskurve.** Sie fängt klein an,
 
 Aber auch diese Simulation ist *immer noch* falsch. Wir übersehen die Tatsache, dass <icon i></icon> ansteckenden Menschen schließlich aufhören, ansteckend zu sein, entweder indem sie 1) sich erholen, 2) sich mit Lungenschäden "erholen" oder 3) sterben.
 
-Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon> würden. (Denken Sie aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> nicht wieder infiziert werden können, und lassen Sie uns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
+Der Einfachheit halber wollen wir so tun, als ob alle <icon i></icon> ansteckenden Menschen wieder genesen <icon r></icon> würden. (Denkt aber daran, dass in Wirklichkeit einige von ihnen sterben.) <icon r></icon> können nicht wieder infiziert werden, lasst ns - *vorläufig!* - so tun, als ob sie ein Leben lang immun bleiben.
 
-Mit COVID-19 sind Sie *im Durchschnitt* schätzungsweise 10 Tage lang  ansteckend.[^infectiousness] Das bedeutet, dass manche <icon i></icon> Menschen sich früher als nach 10 Tagen erholen, andere später. **So sieht das bei einer Simulation aus, die mit 100 % <icon i></icon> *beginnt*:**
+Es wird angenommen, dass sie mit COVID-19 *im Durchschnitt* 10 Tage lang ansteckend sind.[^infectiousness] Das bedeutet, dass manche <icon i></icon> Menschen sich früher als nach 10 Tagen erholen, andere später. **So sieht das bei einer Simulation aus, die mit 100 % <icon i></icon> *beginnt*:**
```

Comment:
```suggestion
Es wird angenommen, dass Personen mit COVID-19 *im Durchschnitt* 10 Tage lang ansteckend sind.[^infectiousness] Das bedeutet, dass manche <icon i></icon> Menschen sich früher als nach 10 Tagen erholen, andere später. **So sieht das bei einer Simulation aus, die mit 100 % <icon i></icon> *beginnt*:**
```
"Sie" wollten wir grundsätzlich mit "du" ersetzen. 
Das "you" im Original hier ist eher als "man" gemeint.
Hab "Personen" statt "man" bevorzugt.